Bottom: 6851eb4e8e4cc6ffbae8ebabd5ac4550dfb8d1b0
Top:    578f0bc048b38d9a175626771d313482e8d0d6c0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:37:56 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index aa7d978..d3bedb3 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1592,6 +1592,12 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+#if 1
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+#endif
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
